### ISSUE TITLE
Sync or Share all/particular apps across devices?

### ISSUE THREAD

**Issue by rafnov** [Droid-ify/client#797]
Is it possible to make sync feature for Droid-ify that will sync devices on demand?
The thing I need is what Google store can do: I log in on multiple devices with the same account and browsing through installed apps on master device I can tell store to install the needed ones on slaves.
Exporting / importing apps via list is not what I need, at least for now.

**Comment by rafnov** [comment]
Export / import reps/favs/settings works well, but requires additional app.
Better would be making what original F-droid client supports:  share nearby. Why sync when we can  SHARE them all: settings/favs/repos/files?
 What you think?
Advantages:
1. No additional app required
2. No additional downloads required

**Comment by Iamlooker** [comment]
Sync/share is a very large and demanding feature, I do not have the bandwidth support it for now. Sorry

**Comment by beedell-roke** [comment]
@Iamlooker, I agree. Would synchronising the preferences to a file in `/storage/emulated/` be acceptable, instead? I ask because that would allow the user to configure SyncThing to synchronise it between devices.

**Comment by rafnov** [comment]
Afaik, you can export/import favs, settings and repos to Download (or other manually set directory), then you can SyncThing or send manually via LocalSend. 

Amazing, how the time passes by... My original question from September 2024...

**Comment by beedell-roke** [comment]
@rafnov, it doesn't automatically import from a directory, though, so it's not much more convenient.

**Comment by rafnov** [comment]
Less automation, more conscious control, what else can I say 🤫
